src/components/ArticlePreview.jsx

```diff
@@ -4,34 +4,40 @@ import { Link } from 'react-router';
 export default class ArticlePreview extends React.Component {
 
   render () {
-    var article = this.props.article;
+    let article = this.props.article;
 
     // Render all contributing authors
     function renderAuthors() {
       // Return nothing if no authors listed
-      return (article.authors || []).map((author) => {
-          <div>
-            <Link to={'/author/' + author.slug}>
-              {author.name}
-            </Link>
-          </div>
-      });
+      let data = [];
+      if (article.authors){
+        for (var author in article.authors){
+          if (article.authors.hasOwnProperty(author)) {
```

Comment:
Why do you need to do the if check again? Also I think this entire loop with `_.map` which allows you to map on arrays, and you can use a similar loop as the one above.